config/ola.m4

```diff
@@ -100,9 +100,30 @@ AM_CONDITIONAL(BUILD_OLA_PROTOC_PLUGIN, test "${with_ola_protoc_plugin}" == "no"
 ])
 
 
-# PLUGIN_SUPPORT(plugin, conditional, prerequisites_found)
-# Build the specified plugin, unless it was disabled at configure time.
-# -----------------------------------------------------------------------------
+# PLUGIN_SUPPORT(plugin_key, conditional, prerequisites_found)
+#
+# Build the specified plugin if requested and its dependencies are met.
+#
+# Each plugin specified with PLUGIN_SUPPORT gets its own configure switch named
+# --enable_${plugin_key}. The three acceptable values the user can give are:
+#
+#  - auto (default): Build the plugin if its dependencies are met, otherwise
```

Comment:
I don't think we should ever get the user to give this option, it should just be the default.